cmd/goreleaser/internal/configure.go

```diff
@@ -67,26 +72,42 @@ func Builds(dist string) []config.Build {
 // Build configures a goreleaser build.
 // https://goreleaser.com/customization/build/
 func Build(dist string) config.Build {
+	var goos []string
+	var archs []string
+	var ignore []config.IgnoredBuild
+	var armVersions []string
+	flags := []string{"-trimpath"}
+	if dist == "otelcol-k8s" {
```

Comment:
Can you add this name as a const.

Reply:
set flags as a const set to `-trimpath` or set a const equal to `otelcol-k8s`?

Reply:
const equal to `otelcol-k8s`.  Lets use constants for all distro names in this generation process.